### PR TITLE
Reflect 4.6.4 release

### PR DESCRIPTION
**Pull request by mkleen**
## Checklist

 - [x] Added an entry in `CHANGES.txt` for user facing changes
 - [x] Updated documentation & `sql_features` table for user facing changes
 - [x] Touched code is covered by tests
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [x] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

File: docs/appendices/release-notes/unreleased.rst [hunk 1]
```diff
@@ -114,40 +114,6 @@ Changes
...
-  before.
-
-- Fixed an issue that caused ``INSERT INTO`` statements to fail on partitioned
-  tables where the partitioned column is generated and the column and value are
-  provided in the statement.
-
- - Fixed an issue that caused ``NullPointerException`` when inserting into
-   previously altered tables that were partitioned and had generated columns.
-
 - Fixed an issue that caused ``UNION ALL`` statements to succeed or throw
```
Comment (mkleen) [inline docs/appendices/release-notes/unreleased.rst:117]: This was not backported, is this intentionally ?

Comment (seut) [reply docs/appendices/release-notes/unreleased.rst:117]: @jeeminso ^^ Fix only related to master?

**Review by seut — APPROVED** [review]

File: docs/appendices/release-notes/unreleased.rst (hunk 1 above)
Comment (jeeminso) [reply docs/appendices/release-notes/unreleased.rst:117]: @mkleen, @seut sorry I think this is supposed to be backported. I was not aware of the process to tag bug fixes

Comment (mkleen) [reply docs/appendices/release-notes/unreleased.rst:117]: ok, no problem. @Seut how should we handle that ?

Comment (seut) [reply docs/appendices/release-notes/unreleased.rst:117]: @jeeminso @mkleen ok so lets backport it then ;)

**Pull request merged**